repository/SmalltalkCI-Pharo-Core.package/SmalltalkCIPharo7.class/class/isPlatformCompatible.st

```diff
@@ -0,0 +1,7 @@
+compatibility
+isPlatformCompatible
+	Smalltalk at: #'SystemVersion' ifPresent: [ :cl |
+		| versionString |
+		versionString := cl current version.
+		^ (versionString beginsWith: 'Pharo7.0') ].
```

Comment:
This will only work for exactly `Pharo7.0` and not for, for example, `Pharo7.1` in the future.
Why not `... beginsWith: 'Pharo7 ...`?